services/frontend-service/pkg/handler/releasetrain.go

```diff
@@ -91,3 +87,42 @@ func (s Server) handleReleaseTrain(w http.ResponseWriter, req *http.Request, tar
 	}
 	w.Write(json) //nolint:errcheck
 }
+
+func (s Server) handleReleaseTrainPrognosis(w http.ResponseWriter, req *http.Request, target string) {
+	if req.Method != http.MethodGet {
+		http.Error(w, fmt.Sprintf("releasetrain prognosis only accepts method GET, got: '%s'", req.Method), http.StatusMethodNotAllowed)
+		return
+	}
+
+	queryParams := req.URL.Query()
+	teamParam := queryParams.Get("team")
+
+	response, err := s.ReleaseTrainPrognosisClient.GetReleaseTrainPrognosis(req.Context(), &api.ReleaseTrainRequest{
+		Target:     target,
+		CommitHash: "",
+		Team:       teamParam,
+	})
+
+	if err != nil {
+		handleGRPCError(req.Context(), w, err)
+		return
+	}
+	json, err := json.Marshal(response.EnvsPrognoses)
+	if err != nil {
+		w.Write([]byte(fmt.Sprintf("error while serializing response, error: %v", err.Error())))
+		return
+	}
+	w.Write(json) //nolint:errcheck
+}
+
+func (s Server) handleReleaseTrain(w http.ResponseWriter, req *http.Request, target, tail string) {
+	switch tail {
+	case "/":
+		s.handleReleaseTrainExecution(w, req, target)
+	case "/prognosis":
+		s.handleReleaseTrainPrognosis(w, req, target)
```

Comment:
the prognosis endpoint must be part of the new `/api/` prefix, so that it respects the flag `ApiEnableDespiteNoAuth`.
See https://github.com/freiheit-com/kuberpult/blob/6198f9c0fd22bf71b035501d1947e17205666183/services/frontend-service/pkg/handler/handle.go#L53

Reply:
yes, looks good